src/style.css

```diff
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center center;
+}
+
+.stars--1 .stars__star:nth-child(-n+1),
```

Comment:
You missed the .stars--0 selector. Add .stars--0 .stars__star:nth-child(-n+0), to the list of selectors to cover all the cases from 0 to 5 active stars.